tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,20 +1,85 @@
-nxfuse
-======
+# How to make smartfs user image for TizenRT
+
+## Contents
+> [How to Enable SmartFS user image](#how-to-enable-smartfs-user-image)  
+> [About NxFuse tool](#about-nxfuse-tool)  
+> [How to use nxfuse](#how-to-use-nxfuse)  
+> [Formatting and Reformatting](#formatting-and-reformatting)  
+
+## How to Enable SmartFS user image
+SMARTFS file system(CONFIG_FS_SMARTFS) needs to be enabled in order to use SMARTFS image.
+
+Enable FS_SMARTFS config through menuconfig at *os* folder.
+
+### Configurations
+Enable below configurations as followings
+
+```
+->BuildSetup
+	->Binary Output Formats
+		-> Enable "Make smartfs binary image" (=y)
```

Comment:
This config (CONFIG_SMARTFS_IMAGE) is not used anywhere. Could you let me know whether this is necessary or not?

Reply:
will be used in future for case based builds. we have such flags ROMFS & SMARTFS support already. So this would also be needed for future purpose

Reply:
Could you let me know your plan? Because I have no idea for that.